process-engine/management_api_client
# ⬆️ Update management_api_contracts

## What did you change?

This PR updates the management_api_contracts

## How can others test the changes?

- Run
```bash
npm install
npm list @process-engine/management_api_contracts
```
- The output should contain
```
@process-engine/management_api_contracts@0.7.1-a5baf823-b2
```


## PR-Checklist

Please check the boxes in this list after submitting your PR:
